sql/hive/benchmarks/OrcReadBenchmark-results.txt

```diff
@@ -3,220 +3,220 @@ SQL Single Numeric Column Scan
 ================================================================================================
 
 OpenJDK 64-Bit Server VM 1.8.0_312-b07 on Linux 5.11.0-1022-azure
-Intel(R) Xeon(R) Platinum 8272CL CPU @ 2.60GHz
+Intel(R) Xeon(R) CPU E5-2673 v4 @ 2.30GHz
```

Comment:
`OrcReadBenchmark-jdk11-results.txt` and `OrcReadBenchmark-jdk17-results.txt` should be updated at the same time